packages/utilities/dom-animation/tsconfig.json

```
@@ -0,0 +1,5 @@
+
+{
+  "extends": "../../../tsconfig.json",
+  "include": ["src", "index.ts"]
+}
```

These seems to be an extra blank line.

```suggestion
{
  "extends": "../../../tsconfig.json",
  "include": ["src", "index.ts"]
}
```